browsers/firefox_android.json

```diff
@@ -378,7 +378,7 @@
         "60": {
           "release_date": "2018-05-09",
           "release_notes": "https://developer.mozilla.org/docs/Mozilla/Firefox/Releases/60",
-          "status": "esr",
+          "status": "retired",
```

Comment:
This ESR change I think would be best reviewed by @ddbeck.

Reply:
I've separated this into #6627!